1.8/administration/overlay-networks/ip-per-container.md

```diff
@@ -69,7 +69,7 @@ In the above example, we have defined two virtual networks. The virtual network
 
 # Retrieving virtual network state
 
-After DC/OS installation is complete, you can query the virtual network configuration using the `https://leader.mesos/overlay-master/state` endpoint from within the cluster. The `network` key at the bottom lists the current overlay configuration and the `agents` key is a list showing how overlays are split across the Mesos agents. The following shows the network state when there is a single overlay in the cluster named `dcos`.
+After DC/OS installation is complete, you can query the virtual network configuration using the `https://<master-IP>/mesos/overlay-master/state` endpoint from within the cluster. The `network` key at the bottom lists the current overlay configuration and the `agents` key is a list showing how overlays are split across the Mesos agents. The following shows the network state when there is a single overlay in the cluster named `dcos`.
```

Comment:
We should get rid of "from within the cluster". 

you can query the virtual network configuration using the `https://<master-IP>/mesos/overlay-master/state` endpoint

OR 

you can query the virtual network configuration using the `https://<master-IP>/mesos/overlay-master/state` endpoint from your browser